test-three/side-menu-app/src/components/SimpleMenuItem.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+// icons
+import SettingsOutlinedIcon from '@material-ui/icons/SettingsOutlined';
+import AssessmentOutlinedIcon from '@material-ui/icons/AssessmentOutlined';
+import BrushOutlinedIcon from '@material-ui/icons/BrushOutlined';
+import TocOutlinedIcon from '@material-ui/icons/TocOutlined';
+import '../styles/style.css';
+
+export default function SimpleMenuItem({ name, subItems }) {
+
+  return ( 
+    <>
+      <ListItem button style={{ paddingLeft: 18 }}>
+        <ListItemIcon>
+          {name === "Account" && <SettingsOutlinedIcon className="leftIcon" />}
```

Comment:
Do you think there may be a better way of doing this rather than hard coding it in the component itself?

Reply:
Yes, I can create a new component that does all the handling of the icons. This way there won't be any repetition between different component and make the code much more cleaner. In addition, we can add more icons to it in the future without disturbing the code. Please feel free to have a look at updated files.